Makefile

```diff
@@ -667,6 +672,15 @@ lint-dockerfiles:
 lint-fix: $(GOLANGCI_LINT) ## Lint the codebase and run auto-fixers if supported by the linter
 	GOLANGCI_LINT_EXTRA_ARGS=--fix $(MAKE) lint
 
+.PHONY: lint-api
+lint-api: GOLANGCI_LINT_EXTRA_ARGS?=--new-from-rev=main
```

Comment:
What happens once this Makefile will end up on a release branch?

Reply:
I think it may be simpler to not use `--new-from-rev`, but exclusions instead 🤔 We may have to build up that list then, but this seems to be the easier approach and always works on any branch or commit then.

Reply:
Is there an env var that would expose the branch name of the base branch? I know on prow you have `PULL_BASE_SHA` but I don't know GH actions well

Reply:
Okay with the decision to not use --new-from-rev for now this finding became irrelevant